addons/account_qr_code_sepa/models/res_bank.py

```diff
@@ -27,6 +28,24 @@ def _get_qr_code_url(self, qr_method, amount, currency, debtor_partner, free_com
                 comment[:141],                                          # Remittance Information (Unstructured) (can't be set if there is a structured one)
                 '',                                                     # Beneficiary to Originator Information
             ]
+            return qr_code_vals
+        return super()._get_qr_vals(qr_method, amount, currency, debtor_partner, free_communication, structured_communication)
+
+    def _get_qr_code_base64(self, qr_method, amount, currency, debtor_partner, free_communication, structured_communication):
```

Comment:
Is there any sense in overriding _get_qr_code_base64 in account_qr_code_sepa ? Why not putting all the logic in account, and have get_qr_vals hande the condition with the qr_method ?

Reply:
My bad. I indeed have not tried the SEPA flow, too focused on l10n_ch.
I will try the SEPA flow this time.

Reply:
See my other comment ; with it we could remove the override and only keep _get_qr_code_url in account, by overriding another function :)